### PR TITLE
refactor: rename interfaces parsed and range 

### PR DESCRIPTION
**Pull request by RahulGautamSingh**
## Changes:

- Rename parsed to Pep440Version

## Context:

- Fixes: #117

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { parsed } from "./lib/shared";
+import { Pep440Version } from "./lib/shared";
```
Comment (viceice) [inline index.d.ts:1]:
```suggestion
import type { Pep440Version } from "./lib/shared";
```

File: lib/specifier.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import type { parsed, Range } from "./shared";
+import type { Pep440Version, Range } from "./shared";
```
Comment (viceice) [inline lib/specifier.d.ts:1]: Maybe we should also rename `Range` to `Pep440Range` ?

@rarkins WDYT?

Comment (rarkins) [reply lib/specifier.d.ts:1]: Good idea

Comment (rarkins) [reply lib/specifier.d.ts:1]: Or `Pep440Constraint` ?

File: index.d.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
-import { parsed } from "./lib/shared";
+import type { Pep440Version } from "./lib/shared";
```
Comment (viceice) [inline index.d.ts:1]:
```suggestion
import type { Pep440Constraint, Pep440Version } from "./lib/shared";
```

File: index.d.ts [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 import type { SemVer } from "semver";
```
Comment (viceice) [inline index.d.ts:3]:
```suggestion

export type { Pep440Constraint, Pep440Version };

```
For easier type usage in renovate.

File: lib/specifier.d.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import type { parsed, Range } from "./shared";
+import type { Pep440Version, Pep440Constraint } from "./shared";
 
 export const RANGE_PATTERN: string;
```
Comment (viceice) [inline lib/specifier.d.ts:3]: This is also used by renovate, so we should export it from index too in a separate PR

**Review by viceice — APPROVED** [review]

**Pull request merged**